app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
```

Comment:
The `Person` class is being used as a data holder and a registry at the same time. This can make the code harder to understand and maintain. Consider separating these responsibilities into two classes: one for holding the data (`Person`) and another for managing the registry of persons.